src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -347,4 +347,33 @@ struct SvgLoaderData
     bool result = false;
 };
 
+#ifdef THORVG_LOG_ENABLED
+inline string nodeTypeToString(SvgNodeType type)
```

Comment:
does it necessary to use inline? leave to compiler if possible
Also, if it uses only in _inefficientNodeCheck(), you can move it in one same place.

#ifdef THORVG_LOG_ENABLED
static string _nodeTypeToString()
{
...
}
static void _inefficientNodeCheck()
{
...
}
#endif

Reply:
Removed inline
and _nodeTypeToString is also used in XmlParser.cpp. Therefore, it cannot be moved to tvgSvgLoade.